vividus-engine/src/main/java/org/vividus/expression/AbstractExpressionProcessor.java

```diff
@@ -35,11 +35,11 @@ public Optional<T> execute(String expression)
         Matcher expressionMatcher = pattern.matcher(expression);
         if (expressionMatcher.find())
```

Comment:
I think it would be more consistent if we initialize ExpressionArgumentMatcher at line 35 and all subsequent calls will be done on the wrapper including find

Reply:
doubtful, we'll need argument matcher only if we find expression

Reply:
then its better to rename the class because matcher in its name seems too wide, I believe it should be more specific